pkg/controller/plan/kubevirt.go

```diff
@@ -292,7 +292,12 @@ func (r *KubeVirt) EnsureVM(vm *plan.VMStatus) (err error) {
 		err = liberr.Wrap(err)
 		return
 	}
-	for _, dv := range dvs.Items {
+	pvcs, err := r.getPVCs(vm)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	/*for _, dv := range dvs.Items {
```

Comment:
I wonder if we wish to have ownership of the VM to the DV and not only to the PVC.

Reply:
answering myself - since the dv is deleted by the garbage collector, we don't need it.